bin/download_secrets.rb

```diff
@@ -1 +0,0 @@
-../config/deploy/docker/assets/download_secrets.rb
```

Comment:
Moved this file to `bin/` (instead of symlinking) to simplify pathing logic in `require_relative`.